main_test.go

```diff
@@ -40,14 +39,18 @@ func TestJoin(t *testing.T) {
 		os.Getenv("REPLY_MESSAGE"),
 	)
 	api := &API{handler}
+	mux := http.NewServeMux()
+	api.registHandler(mux)
 
 	cases := []struct {
 		name     string
+		method   string
 		endpoint string
 		expect   int
 		handler  http.Handler
```

Comment:
need?

Reply:
yes.
if the method is `http.MethodPost`, the request should contain body.
if the method is `http.MeghodGet`, the request should not contain this.

Reply:
I want to switch based on http Mehod in `testNewRequest`